extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -190,8 +191,9 @@ export class AmpA4A extends AMP.BaseElement {
 
   /**
    * @param {!Element} element
+   * @param {number} upgradeDelayMs
    */
-  constructor(element) {
+  constructor(element, upgradeDelayMs) {
```

Comment:
Do we really need to modify the constructor for this? If we are okay with sending the ping after construction, we can do something like:

```
return ctorPromise
    .then(ctor => {
        const delay = getNow() - startTime;
        const a4a = new ctor(this.element);
        a4a.passUpgradeDelay(delay);
        return a4a;
    });
```
And have passUpgradeDelay emit the ping.